qa/workunits/test_telemetry_reef.sh

```diff
@@ -31,7 +31,7 @@ REPORTED_COLLECTIONS=$(ceph telemetry collection ls)
 NUM_REPORTED_COLLECTIONS=$(echo "$REPORTED_COLLECTIONS" | awk '/^NAME/ {flag=1; next} flag' | wc -l)
 KNOWN_COLLECTIONS=("basic_base" "basic_mds_metadata" "basic_pool_options_bluestore"
                    "basic_pool_usage" "basic_rook_v01" "basic_usage_by_class" "crash_base"
-		   "device_base" "ident_base" "perf_memory_metrics" "perf_perf")
+		   "device_base" "ident_base" "perf_memory_metrics" "perf_perf" "basic_stretch_cluster")
```

Comment:
The "basic_stretch_cluster" collection hasn't been added to reef yet, so we should not update this workunit yet. After then change has been backported to reef, then this workunit should be updated.

Reply:
apologies I missed this! thank for catching this :)